src/web.rs

```diff
@@ -497,6 +497,25 @@ impl WebCanvasElement
         )
     }
 
+    pub fn get_webgl2_context_from_callback<V, CB>(
+        viewport_size_pixels: V,
+        cb: CB
```

Comment:
Is there any reason to pass a callback here, rather than just passing the `web_sys::WebGl2RenderingContext` directly?

E.g. the signature could be:

```rust
pub fn use_existing_webgl2_context(
        viewport_size_pixels: impl Into<UVec2>,
        context: web_sys::WebGl2RenderingContext
)  -> Result<GLRenderer, BacktraceError<GLRendererCreationError>>
{ ... }
```

Reply:
No particular reason. Will change it.